packages/widgetbook_annotation/README.md

```diff
@@ -0,0 +1,180 @@
+[![Discord](https://img.shields.io/discord/879618555560218625?color=blue&style=flat-square)](https://discord.com/invite/zT4AMStAJA)
+[![style: very good analysis](https://img.shields.io/badge/style-very_good_analysis-B22C89.svg?style=flat-square)](https://pub.dev/packages/very_good_analysis)
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/widgetbook/widgetbook/ci?style=flat-square)
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/widgetbook/widgetbook/ci?label=test&style=flat-square)
+
+
+# Introduction 
+
+This package contains annotations for [package:widgetbook_generator](https://pub.dev/packages/widgetbook_generator) with which the generator will create the Widgetbook defined in [package:widgetbook](https://pub.dev/packages/widgetbook). Therefore, this package is a part of making [package:widgetbook](https://pub.dev/packages/widgetbook) easier to maintain. Furthermore, setting up and maintaining [package:widgetbook](https://pub.dev/packages/widgetbook) is simplified by code generation. 
+
+# Installing this package
+This package requires the following dependencies: 
```

Comment:
following what dependencies? Below you write about "dev dependencies". Or is it always used like this?